modules/developer_manual/pages/core/acceptance-tests.adoc

```diff
@@ -240,14 +240,23 @@ sudo -u www-data $installation_path/occ maintenance:install \
   --database-pass=` --admin-user='admin' --admin-pass='admin'
 ----
 
+=== Types of Acceptance Tests
+
+
+There are 3 types of acceptance tests; API, CLI and webUI. API tests test the ownCloud public APIs.
```

Comment:
This is a little nitpicky, perhaps, but listing the description of the three types of tests as a bulleted list should increase readability.

Reply:
Great to hear @phil-davis. Thanks.

Reply:
Um, did I miss something, as the text hasn't changed.

Reply:
Sad, I guess I pushed too quickly and did not look at the message. The changes will be sitting in the office. I will have to push them in the next days when I go by.